### PR TITLE
#4175 [Manager] Failed to start topic when there are not enough brokers

### PR DESCRIPTION
**Pull request by konekoya**
### Checklist
- [x] [Review Contribution Guidelines](https://ohara.readthedocs.io/en/latest/contributing.html)
- [x] Add Reviewers (see [Authors](https://github.com/oharastream/ohara#ohara-team))
- [x] Add Assignees (of course, you are the assignee by default!)
- [x] DON'T set a Milestone
- [x] DON'T add any labels (PR labels will be added automatically by Jenkins)
- [x] Link to an issue(You can do so by using the required by keyword: required by #4175)
- [x] Add unit tests (or please explain why this PR is not worth having new tests)
- [x] Pass QA (Sometimes you may encounter an existing flaky test which would obstruct you from getting QA passed. If the failed tests are unrelated, you can add a comment in the PR thread)

### CODE REVIEW

File: ohara-manager/client/src/utils/validate.js [hunk 1]
```diff
@@ -39,11 +39,15 @@ export const maxLength = max => value => {
 };
 
 export const minNumber = min => value => {
-  return value >= min ? undefined : `The value must be greater than ${min}`;
+  return value >= min
+    ? undefined
+    : `The value must be greater than or equal to ${min}`;
```
Comment (konekoya) [inline ohara-manager/client/src/utils/validate.js]: Well, the message is not correct...

Comment (konekoya) [reply ohara-manager/client/src/utils/validate.js]: This affect many things, I will file another issue to fix this

**Comment by konekoya** [comment]
retry manager-ut

File: ohara-manager/client/src/context/broker/brokerActions.js [hunk 2]
```diff
@@ -50,7 +50,7 @@ export const createActions = context => {
         const startRes = await brokerApi.start(values.name);
         // after created, we need the "settings" and "stageSettings" from creation payload
         // to decide workspace is dirties or not
-        const data = { ...startRes, ...createRes };
+        const data = { ...createRes, ...startRes };
```
Comment (konekoya) [inline ohara-manager/client/src/context/broker/brokerActions.js]: This is where the `aliveNodes` are override... Thanks for the help @saivirtue and @wu87988622

Comment (konekoya) [reply ohara-manager/client/src/context/broker/brokerActions.js]: Turns out there's nothing wrong with our wait logic 😅

File: ohara-manager/client/src/components/Pipeline/Toolbox/ToolboxHooks.js [hunk 3]
```diff
@@ -37,10 +37,15 @@ export const useTopics = () => {
...
     }))
     .filter(
-      // Pipeline-only topics are hidden from Toolbox
-      topic => topic.isShared || topic.name === 'Pipeline Only',
+      // Some type of topics are hidden from the Toolbox, this including:
+      // 1. Pipeline-only topics
+      // 2. Shared topics that are not started yet (without any status)
+      topic =>
+        topic.name === 'Pipeline Only' ||
+        (topic.isShared && topic.status !== undefined),
```
Comment (eechih) [inline ohara-manager/client/src/components/Pipeline/Toolbox/ToolboxHooks.js]: 👍

**Comment by konekoya** [comment]
retry manager-e2e

**Pull request merged**